bin/build

```diff
@@ -22,18 +22,21 @@ fi
 
 echo "Building secretless-broker:$TAG Docker image"
 docker build -t "secretless-broker:${TAG}" \
+             -t "secretless-broker:latest" \
```

Comment:
Can we add comments here to say which downstream dependencies depend on the local latest images? It's not clear that those dependencies exist from this file, which led to me deleting things and breaking them once.

Reply:
Done